### PR TITLE
Add Kinesis Data Streams server object

### PR DESCRIPTION
**Pull request by maxjoh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -485,6 +485,15 @@ servers:
 | format    | `string` | Format of files, such as `parquet`, `delta`, `json`, `csv`                                                       |
 | delimiter | `string` | (Only for format = `json`), how multiple json documents are delimited within one file, e.g., `new_line`, `array` |
 
+#### AWS Kinesis Data Streams Server Object
+
+| Field  | Type     | Description                                                               |
+|--------|----------|---------------------------------------------------------------------------|
+| type   | `string` | `kinesis`                                                                 |
+| name   | `string` | The name of the Kinesis data stream.                                      |
```
Comment (MattiasMTS) [inline README.md:493]: perhaps worth using the arn here instead of the name or whatya think?

File: README.md [hunk 2]
```diff
@@ -485,6 +485,15 @@ servers:
...
+| region | `string` | AWS region, e.g., `eu-west-1`.                                            |
+| format | `string` | The format of the records. Examples: json, avro, protobuf.                |
```
Comment (MattiasMTS) [inline README.md:495]: looking at AWS Glue example, let's add account as well:
`| account  | `string` | REQUIRED. The AWS account, e.g., `1234-5678-9012`          |`

**Review by MattiasMTS — COMMENTED** [review]
Left some comments.

Did you try this E2E with an existing kinesis stream in e.g. automation-dev? The realtime analytics kinesis stream perhaps?

File: README.md (hunk 1 above)
Comment (maxjoh) [reply README.md:493]: To connect a client to kinesis data stream you only need the name, and optionally the region. I haven't seen a client needing the arn at least.

And looking at the other implementations that's how I see they want this to work. You supply the information needed to consume the data.

Comment (MattiasMTS) [reply README.md:493]: Gotcha! Let's go with that then 😋

File: README.md (hunk 2 above)
Comment (maxjoh) [reply README.md:495]: same here, account is not needed to connect to a kinesis stream. But yeah looking at all server objects it looks like they are not all following the same pattern, so maybe. Does it help with having the account id? I don't think so tbh

**Comment by maxjoh** [comment]
> Left some comments.
> 
> Did you try this E2E with an existing kinesis stream in e.g. automation-dev? The realtime analytics kinesis stream perhaps?

Hm, what do you mean?

File: README.md (hunk 2 above)
Comment (MattiasMTS) [reply README.md:495]: Probably _not_ since when you would call this tool you are hopefully already in SSO environment and need only the name. Nice!

**Review by MattiasMTS — APPROVED** [review]

**Pull request merged**